docs/bigtable.txt

```diff
@@ -1,6 +1,5 @@
 [[bigtable]]
-Google Cloud Bigtable
----------------------
+== Google Cloud Bigtable
```

Comment:
Since this is the main title for the entire page, should this be h1, i.e., `=`, and the other headings up-leveled one level?